notifications_utils/field.py

```diff
@@ -48,6 +49,18 @@ def __repr__(self):
         return "Placeholder({})".format(self.body)
 
 
+def get_sanitizer(method) -> Callable:
+    if method == "strip":
```

Comment:
perhaps this is better as a dictionary and accessing it via key rather than if else statements. I agree with the separate function. Perhaps this is better as a static function within the class though ?

Reply:
Okay I will move it inside the class